app/scripts/models/user.js

```diff
@@ -40,14 +41,18 @@ define(function (require, exports, module) {
       // consumers so that they don't have to refetch the account's
       // ephemeral data, e.g. OAuth access tokens.
       this._cachedSignedInAccount = null;
+
+      this._flow = new FlowModel(options);
```

Comment:
Passing options straight to FlowModel doesn't make me nervous so much as it doesn't really give me insight into what the FlowModel requires. Can you use [_.pick](http://underscorejs.org/#pick) to list the keys that are depended upon?